drivers/bluetooth/hci/spi.c

```diff
@@ -258,9 +258,10 @@ static int configure_cs(void)
 #ifdef GPIO_CS_PIN
 	static struct spi_cs_control spi_conf_cs;
 
-	spi_conf_cs.gpio_pin = GPIO_CS_PIN,
+	spi_conf_cs.gpio_pin = GPIO_CS_PIN;
 	spi_conf_cs.gpio_dev = device_get_binding(
 		DT_INST_SPI_DEV_CS_GPIOS_LABEL(0));
+	spi_conf_cs.gpio_dt_flags = GPIO_CS_FLAGS;
```

Comment:
Move this above the assignment to gpio_dev, since the error check of that is below, now this line is squeezed in between those.

Reply:
Done ! 👍